Sorry @ssam18 linter again :( 

```
  src/analytics/utils.test.ts:18:41 - error TS2352: Conversion of type '{ addEvent: jest.Mock<any, any, any>; }' to type 'EchoSrv' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
    Type '{ addEvent: Mock<any, any, any>; }' is missing the following properties from type 'EchoSrv': flush, addBackend
  
  18 jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as ReturnType<typeof getEchoSrv>);
                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
  src/analytics/utils.test.ts:23:45 - error TS2352: Conversion of type '{ addEvent: jest.Mock<any, any, any>; }' to type 'EchoSrv' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
    Type '{ addEvent: Mock<any, any, any>; }' is missing the following properties from type 'EchoSrv': flush, addBackend
  
  23     jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as ReturnType<typeof getEchoSrv>);
  ```

--- a/packages/grafana-runtime/src/analytics/utils.test.ts
+++ b/packages/grafana-runtime/src/analytics/utils.test.ts
@@ -15,12 +15,12 @@ jest.mock('../config', () => ({
 }));
 
 const mockAddEvent = jest.fn();
-jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as ReturnType<typeof getEchoSrv>);
+jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as unknown as ReturnType<typeof getEchoSrv>);
 
 describe('reportPageview', () => {
   beforeEach(() => {
     jest.clearAllMocks();
-    jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as ReturnType<typeof getEchoSrv>);
+    jest.mocked(getEchoSrv).mockReturnValue({ addEvent: mockAddEvent } as unknown as ReturnType<typeof getEchoSrv>);
     config.appSubUrl = '';
   });
 

